pandas/tests/extension/json/array.py

```diff
@@ -27,12 +27,7 @@
 class JSONDtype(ExtensionDtype):
     type = abc.Mapping
     name = 'json'
-
-    try:
-        na_value = UserDict()
-    except AttributeError:
-        # source compatibility with Py2.
-        na_value = {}
+    na_value = UserDict()
```

Comment:
can this just be a dict or OrderedDict?

Reply:
From this comment in same file, using `UserDict` is intentional in order to treat dict as a scalar.

>We currently store lists of UserDicts. Pandas has a few places
internally that specifically check for dicts, and does non-scalar things
in that case. We *want* the dictionaries to be treated as scalars, so we
hack around pandas by using UserDicts.